docs/src/guides/symfony/deploy/_index.md

```diff
@@ -4,9 +4,65 @@ sidebarTitle: "Get started"
 weight: -110
 layout: single
 description: |
-    Create a Platform.sh account, download a few tools, and prepare to deploy Symfony.
+    See how to get started deploying Symfony on Platform.sh.
 ---
 
-Symfony is a PHP framework. Platform.sh is the official Symfony PaaS.
+Symfony is a web application framework written in PHP. Platform.sh is the official Symfony PaaS.
 
-Read the [official integration documentation](https://symfony.com/doc/current/cloud/intro.html) to learn more about how to deploy Symfony applications on Platform.sh.
+This guide provides instructions for deploying, and working with, Symfony on Platform.sh.
+It includes examples for working with Symfony on all the major package managers: composer.
+
+{{% guides/symfony/starting-point name="Symfony" templateRepo="platformsh-symfony-template" templateRepoMetadata="platformsh-symfony-template-metadata" template="symfony-6.2.template" initExample=true %}}
+
+{{< note >}}
+
+This guide is written for Symfony 6.2, but should apply almost exactly the same for other versions.
+
+{{< /note >}}
+
+{{% guides/symfony/requirements %}}
+
+{{< note >}}
+
+Note:
+All sample commands in this guide using Symfony CLI `symfony <command>` can be replaced by `platform <command>` as Symfony CLI is an official replicate of the [Platform.sh CLI](/administration/cli/_index.md).
```

Comment:
@flovntp some of the commands have `symfony cloud:` and not just `symfony`. Is there a rule on when that's true? Because I think leaving the `cloud:` part in wouldn't work with the Platform.sh CLI?